### ISSUE TITLE
Support for vue components

### ISSUE THREAD

**Issue by scottsweb** [dword-design/nuxt-content-body-html#57]
Nuxt content allows you to use custom components in your markdown (e.g. `v-image`) and it looks like those components are not rendered as html by this plugin. I am wondering if it is something that could be supported?

**Comment by dword-design** [comment]
Yeah would be nice to have this. I don't know how though because you would have to somehow boot Nuxt so that the components are present and then render them. I currently use `rehype-stringify` to convert the markdown into an HTML string. Would be great to have a PR on this.

**Comment by scottsweb** [comment]
Thanks for the explanation, that does sound like quite a big lift. I have found another approach for now that involves scraping content from the `dist` folder which is working well. Happy to close this if needed.

**Comment by dword-design** [comment]
I think I'll leave it open for now and maybe there is a way to implement it in the future.
